jail-events.yml

```diff
@@ -5,27 +5,9 @@
   connection: local
   become: yes
   vars:
-    env_name: venv
     web_host_name: "events.buildbot.net"
 
   roles:
   - base
-  - role: user
```

Comment:
I was first planning for this host to create a twisted application, and run that in this venv, but it looks like elk might be suffisiant for the whole design. I might add a virtualenv back if logstash http reveals not to be enough (I am not 100% sure yet)